internal/sink/sink.go

```diff
@@ -68,31 +67,32 @@ func NewSink(c Config, logger *zap.Logger, listener ancla.Register) Sink {
 			kafka := &Kafka{
 				id:         l.Registration.CanonicalName,
 				brokerAddr: k.BootstrapServers,
-				topic:      "test",
+				topic:      "quickstart-events",
+				logger:     logger,
 			}
 
+			config := sarama.NewConfig()
 			//TODO: this is basic set up for now - will need to add more options to config
 			//once we know what we are allowing users to send
-			kafka.config.Producer.Return.Successes = true			
-			kafka.config.Producer.RequiredAcks = sarama.WaitForAll
-			kafka.config.Producer.Retry.Max = c.DeliveryRetries //should we be using retryhint for this?
 
+			config.Producer.Return.Successes = true
```

Comment:
I added back in the default config because it kept failing due to certain fields not being set. i'm setting these fields this way based on a tutorial I found: https://youtu.be/j6bqJKxb2w0?si=-htt-I0kgF8RqHMy&t=1384